src/drivers/cmakeFileApiDriver.ts

```diff
@@ -38,7 +38,7 @@ const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 
 const log = logging.createLogger('cmakefileapi-driver');
 /**
- * The CMake driver with FileApi of CMake >= 3.15.0
+ * The CMake driver with FileApi of CMake >= 3.14.0
```

Comment:
Why is this version being changed?

Reply:
The version isn't changed. I think the version in the comment was incorrect for some reason. This is the actual version we check (unmodified) 
![image](https://github.com/microsoft/vscode-cmake-tools/assets/113148726/944db9ae-a436-4a3f-8271-404a6c7da492)

Reply:
Ahh, thanks for pointing that out, thanks! 👍